src/components/Contentful/TestimonialCards.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+	<div>
+		<section-with-background :background-content="background">
+			<template #content>
+				<kv-page-container>
+					<kv-grid class="tw-grid-cols-12 tw-text-center">
+						<h2
+							v-html="testimonialHeadline"
+							class="tw-py-4 tw-col-span-12"
+						>
+						</h2>
+
+						<div
+							v-for="(singleCard, index) in cardData"
+							:key="index"
+							class="tw-col-span-12 md:tw-col-span-6 lg:tw-col-span-6
+							tw-mb-2
+							tw-bg-white tw-rounded"
+						>
+							<div class="tw-pt-4 tw-pb-3">
+								<kv-contentful-img
+									v-if="singleCard.imageUrl"
+									class="tw-rounded-full tw-overflow-hidden"
+									:contentful-src="singleCard.imageUrl"
+									:alt="singleCard.parsedName"
+									:width="64"
+									:height="64"
+									loading="lazy"
+									fallback-format="jpg"
+								/>
+							</div>
+
+							<p
```

Comment:
Sorry, I should have noticed this earlier. Because these components always show testimonials, the semantic markup should be  `blockquote`s.

There's a couple of ways you could mark that up, https://html.spec.whatwg.org/multipage/grouping-content.html#the-blockquote-element

Either,
```
<blockquote>
 <p v-html="singleCard.quote" ...></p>
</blockquote>
<p v-html="singleCard.name" class="tw-text-h3" ...></p>
<p v-html="singleCard.title" class="tw-text-h4" ...></p>
```

Or I kinda like all of it inside a `figure`, like
```
<figure	v-for="(singleCard, index) in cardData" ...>
  img etc,
  <blockquote>
     <p v-html="singleCard.quote" ...></p>
  </blockquote>
  <figcaption>
    <p v-html="singleCard.name" class="tw-text-h3" ...></p>
    <p v-html="singleCard.title" class="tw-text-h4" ...></p>
  </figcaption>
</figure>

Reply:
Bit of a longer article if you're interested: https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/